lib/ts/styles/styles.css

```diff
@@ -234,6 +234,7 @@
     background-position: center;
     transition: all 0.4s;
```

Comment:
is it possible to add unit test for this?

Reply:
it is possible, discussed this with @porcellus and this seem to be a lot of work to just to check that line of css does work

Reply:
The other big issue with the test is that we'd be testing that CSS rules work, but as far as I can tell, there is no way to test something like "what's the cursor like if I hover over this part of the page/button".